projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -1274,6 +1274,7 @@
     %grid-display-container-tr {
         width: 100%;
         overflow: visible;
+        flex: 1;
```

Comment:
The change itself is fine, however since it changes the size of the cell's virtual container you'll also have to change how right pinning positions the cells. It currently sets an incorrect offset so if you pin any column to the right, you'll see that it's not correctly positioned to the right edge of the grid:

<img width="1646" height="720" alt="image" src="https://github.com/user-attachments/assets/63b31d69-dc94-4942-bede-767b08a8fb3a" />